package.json

```diff
@@ -470,5 +470,8 @@
     "uuid": "^8.3.2",
     "validate-npm-package-name": "^3.0.0",
     "vscode-languageclient": "8.0.0-next.2"
-  }
+  },
+  "extensionKind": [
```

Comment:
What is the default value VS Code sets this to, when it's unspecified as we had it previously?

Reply:
[I believe it tries to infer the location automatically](https://code.visualstudio.com/api/advanced-topics/remote-extensions#incorrect-execution-location). One of the [common problems](https://code.visualstudio.com/api/advanced-topics/remote-extensions#incorrect-execution-location) is incorrect execution location; for which setting the `extensionKind` property is a suggested fix.